src/utils/hints/tokensHints.js

```diff
@@ -2,6 +2,7 @@ export const tokensHints = {
   tokenSymbol: 'Non-unique symbol of the token.',
   tokenName: 'Full name of the token.',
   smartContractId: 'Identifier of the Smart Contract that implements the AEX-9 interface.',
+  supplyAmount: 'Initial supply of tokens.',
```

Comment:
What does it mean, do you refer to tokens that have been minted at the moment of contract creation, tokens that have been so far minted or the maximum amount of tokens that can be ever minted?